curated_transformers/tests/tokenization/test_bert_tokenizer.py

```diff
@@ -0,0 +1,243 @@
+import pytest
+import torch
+
+from curated_transformers._compat import has_hf_transformers, transformers
+from curated_transformers.tokenization import PiecesWithIds
+from curated_transformers.tokenization import BertTokenizer
+from curated_transformers.tokenization.bert_tokenizer import BertPreEncoder
+
+from ..util import torch_assertclose
+
+
+@pytest.fixture
+def toy_tokenizer(test_dir):
+    return BertTokenizer.from_files(
+        vocab_path=test_dir / "toy.wordpieces",
+    )
+
+
+@pytest.mark.skipif(not has_hf_transformers, reason="requires huggingface transformers")
+@pytest.mark.slow
+def test_berttokenizer_hf_tokenizer(sample_texts):
```

Comment:
We now have this test in three places, maybe we should factor it out to a function that can be called with the model name? Then if we want to check other things later like padding or attention masks, we could easily add them to all the places.